src/unmanaged/OpenLiveWriter.Ribbon/OpenLiveWriter.Ribbon.vcxproj

```diff
@@ -15,18 +15,19 @@
     <ProjectGuid>{195A60BF-7A4D-42E6-B5F4-FEBC679E19F0}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>OpenLiveWriter.Ribbon</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```

Comment:
Why update the Windows Platform Version? I believe this requires October/1809, but we should still be fine supporting April/1803, correct?

Reply:
Defs, I retargeted to 10.0.17134.0.